src/Giveaway.js

```diff
@@ -96,6 +96,10 @@ class Giveaway extends EventEmitter {
         this.message = null;
     }
 
+    get exemptMembersFunction () {
+        return (this.options.exemptMembers && (typeof this.options.exemptMembers === 'function')) ? new Function(`return ${this.options.exemptMembers}`) : null;
```

Comment:
```suggestion
        return (this.options.exemptMembers && (typeof this.options.exemptMembers === 'function')) ? (this.options.exemptMembers.toString().includes('function anonymous') ? this.options.exemptMembers : eval(this.options.exemptMembers)) : null;
```
```js
const test = '731583387235647498'
exemptMembers: (member) => !member.roles.cache.find(r => r.id === test)
```
Does not work because it would save "test" as "test" inside the DB and not as the assigned value

We can go around this "blockade" with
```js
const test = '\'731583387235647498\''
exemptMembers: new Function('member', `return !member.roles.cache.find(r => r.id === ${test})`)
```
but this works only with my suggested change

To summarize the problem: "Be able to use variable parameters"
_Lmk if there's any better solution_

I tested my change and it works fine
```js
exemptMembers: (member) => !member.roles.cache.find(r => r.id === '731583387235647498')
// or
const test = '\'731583387235647498\''
exemptMembers: new Function('member', `return !member.roles.cache.find(r => r.id === ${test})`)
```
both work with it

Reply:
Or we could do smth different
`exemptMembers: { filter: (member, arguments) => !member.user.username === arguments[0], arguments: ['Nico105']}`
or 
`exemptMembers: { filter: (member, arguments) => !member.user.username === arguments.name, arguments: { name: 'Nico105' } }`
I don't really have a preference but the 2 things in this comment are probably the most user friendly since the one I frist suggested makes everything to a string and like that you could accidently add a error to the filter

Reply:
@Androz2091 Tell me which one is best and I will change the rest of the code and make a pr to this branch 
or if you don't like my last 2 suggestions because it's a object and not a single function anymore then you can also accept my initial suggestion

Reply:
Or we could also just pass in `(member, extraData)` so people can store their arguments in `extraData` but that would maybe not be the best solution cause `extraData` should be for data that the features of the giveaway system itself do not need. So idk about that. As said tell me what you find best.